tests/Integration/UI/NetworkAdminSitesListTest.php

```diff
@@ -80,8 +80,10 @@ public function test_api_key_column_is_present(): void {
 	 * @return void
 	 */
 	public function test_api_key_column_is_correctly_printed(): void {
-		$blog_id_with_api_key    = $this->factory->blog->create();
-		$blog_id_without_api_key = $this->factory->blog->create();
+		$blog_id_with_api_key = $this->factory->blog->create();
```

Comment:
Should we consider renaming this to `$blog_id_with_site_id`?

Reply:
I see your point towards converging to Site ID. However, we're still using `apikey` in the DB representation, thus changing the variable name would make it more confusing.